test/models/messages_test.js

```diff
@@ -174,10 +174,10 @@
             assert.equal(message.getDescription(), 'Updated the group. Title is now \'blerg\'. Bob joined the group.', 'Notes when there are multiple changes to group_updates properties.');
 
             message = messages.add({flags: true});
-            assert.equal(message.getDescription(), 'Secure session ended.');
+            assert.equal(message.getDescription(), i18n('sessionEnded');
```

Comment:
oops, some leprechaun must have stole it
how come the tests still got 100%? :open_mouth::grey_question: